furya/chain.json

```diff
@@ -71,18 +76,34 @@
       {
         "address": "https://rpc.furya.xyz",
         "provider": "Furya Genesis Validator 🐝"
+      },
+      {
+        "address": "https://furya-rpc.synergynodes.com",
+        "provider": "synergynodes"
+      },
+      {
+        "address": "https://furya.rpc.nodeshub.online:443",
+        "provider": "nodeshub"
       }
     ],
     "rest": [
       {
         "address": "https://api.furya.xyz",
         "provider": "Furya Genesis Validator 🐝"
+      },
+      {
+        "address": "https://furya.api.nodeshub.online",
+        "provider": "synergynodes"
+      },
+      {
+        "address": "https://furya.api.nodeshub.online",
+        "provider": "Furya Genesis Validator 🐝"
```

Comment:
Do you mean for the bee emoji to be here?

Reply:
No. Sorry, my bad. Missed that when I copied code blocks.